server/src/test/java/org/spine3/server/BoundedContextShould.java

```diff
@@ -152,7 +151,7 @@ public void return_CommandDispatcher() {
     }
 
     @Test(expected = NullPointerException.class)
-    public void throw_NPE_on_null_CommandRequest() {
+    public void throw_NPE_on_null_Command() {
         // noinspection ConstantConditions
```

Comment:
The problem is that there are no annotations for some of the warnings, and if we stick with annotation-only approach suppressions will cover the whole method. I'll fix it here, though.